DisplayUserInfo-Robert/DisplayUserInfo-Robert/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace DisplayUserInfo_Robert
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello!");
+            Console.WriteLine("What is your first name?");
+
+            string firstName = Console.ReadLine();
+            Console.WriteLine();
+            Console.WriteLine(firstName + ", what is your last name?");
+
+            string lastName = Console.ReadLine();
+            Console.WriteLine();
+            Console.WriteLine(firstName + " " + lastName + "? What a great name!");
+            Console.WriteLine();
+            Console.WriteLine("What is your favorite number?");
+            string number1 = Console.ReadLine();
```

Comment:
This variable could probably have a better name. Either just "number", or something more descriptive of its purpose, like "favoriteNumber"